db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_03_25_112312) do
+ActiveRecord::Schema.define(version: 2022_05_19_185616) do
```

Comment:
Kind of... When I try to run the tests locally, I am getting this error.

```
== 20220519185616 RemoveHintsFromRequest: migrating ===========================
-- remove_column(:requests, :hints)
ERROR:  column "hints" of relation "requests" does not exist
STATEMENT:  ALTER TABLE "requests" DROP COLUMN "hints"
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

PG::UndefinedColumn: ERROR:  column "hints" of relation "requests" does not exist
```
I have gotten past this error in 2 different ways:

1) by adding `if_exists: true` to the `db/migrate/20220519185616_remove_hints_from_request.rb` and running the migrations - doesn't feel like the best way cause the column did exist so it shouldn't throw this error.
2) running `bundle exec rails db:drop db:create db:migrate RAILS_ENV=test`

Both of these update the schema version. The schema version is `2022_03_25_112312`, which corresponds to the `AddAdditionalConsentToContributors` migration `/home/mattwr18/code/100eyes/db/migrate/20220325112312_add_additional_consent_to_contributors.rb`. The schema version was not updated when the `RemoveHintsFromRequest` was added. As you are the one that added this migration, was this an oversight? Or did you purposely not update the schema version for some reason?

I am also able to get past the error by running `bundle exec rails db:setup RAILS_ENV=test` and it does not update the schema version because it loads the schema instead of running the migrations, but I'm curious if/when we update our schema version. Looking at the history of the file, it seems that we are updating it with each new migration that is added. So, I assume this is a change that we actually want to keep, would you prefer it in a separate PR?


 

Every time I run the migrations, even if it is just for the test db, the `db/schema` updates and I typically don't commit it, but in this case I decided to just push it up cause I was tired of rolling back the change.